src/AirshipBindings.NETStandard/AirshipBindings.NETStandard.Android/Airship.cs

```diff
@@ -377,5 +394,17 @@ public bool OnDeepLink(string deepLink)
             handler(deepLink);
             return true;
         }
+
+        public void OnChannelCreated(string channelId)
+        {
+            var handler = onChannelCreation;
+            handler(channelId);
+        }
+
+        public void OnChannelUpdated(string channelId)
+        {
+            var handler = onChannelUpdate;
```

Comment:
i think you need to check if its null

Reply:
idiomatically: `onChannelUpdate?.Invoke(channelID)`